.gitlab-ci.yml

```diff
@@ -222,6 +223,7 @@ Test_N_Build:internal:
 
   only:
   - master
+  - new-cluster-deployments
```

Comment:
See above to use rules

Reply:
But we've never used deploy freeze for client. I remember you introduced it for complex cases like Thoas, where we were sharing the repository with another team, and wanted a fine-grained control over the deployment; but with ensembl-client, we are in full control. Do we really need this?

And if we really need this level of control for deployments to production, why would we need it for deployments to any other environment? Here, you left the comment under `Test_N_Build:internal`

Reply:
That's valid point but only/except are not being actively developed by gitlab and prefer rules over only/except 
https://archives.docs.gitlab.com/15.11/ee/ci/yaml/#only--except